pep_sphinx_extensions/pep_processor/transforms/pep_footer.py

```diff
@@ -31,8 +31,8 @@ def apply(self) -> None:
             if not isinstance(section, nodes.section):
                 continue
             title_words = section[0].astext().lower().split()
-            if "references" in title_words:
-                # Remove references section if there are no displayed
+            if "references" in title_words or "footnotes" in title_words:
+                # Remove references/footnotes section if there are no displayed
```

Comment:
Ideally I'd use a set literal on the line above (`{*section[0].astext().lower().split()}` instead of `{*title_words}`) but GH won't let me put a suggestion on that line and also this line for reasons man has yet to understand.

```suggestion
            if {"references", "footnotes"} & {*title_words}:
                # Remove the references or footnotes sections if there are no displayed
```

Reply:
Yeah, good point—I've manually made the change both here and in `pep2html.py`.

> GH won't let me put a suggestion on that line and also this line for reasons man has yet to understand.

I may or may not be a "man", but the reason, dumb as it is, is that intervening deleted lines that it doesn't like.